pkg/kubectl/describe.go

```diff
@@ -162,6 +162,25 @@ func init() {
 	DefaultObjectDescriber = d
 }
 
+func getStorageClassAnnotation(obj api.ObjectMeta) string {
+	if obj.Annotations[storage.BetaStorageClassAnnotation] != "" {
+		return obj.Annotations[storage.BetaStorageClassAnnotation]
+	}
+	if obj.Annotations[storage.AlphaStorageClassAnnotation] != "" {
+		return obj.Annotations[storage.AlphaStorageClassAnnotation]
+	}
+
+	return ""
+}
+
+func isDefaultAnnotation(obj api.ObjectMeta) string {
```

Comment:
This function might be 'ok-ish' here, but since I want a generic function that one needs to return a `bool`